kotlin-inject-compiler/test/src/test/kotlin/me/tatarka/inject/test/Utils.kt

```diff
@@ -11,4 +14,10 @@ import javax.tools.Diagnostic
 fun Assert<Throwable>.output(): Assert<String> = message().isNotNull()
 
 fun Assert<TestCompilationResult>.warnings(): Assert<String> =
-    prop("warnings") { it.output(Diagnostic.Kind.WARNING) }
\ No newline at end of file
+    prop("warnings") { it.output(Diagnostic.Kind.WARNING) }
+
+internal inline fun <T> assertSuccess(block: () -> T) =
+    assertThat(runCatching(block)).isSuccess()
+
+internal inline fun <T> assertFailure(block: () -> T) =
+    assertThat(runCatching(block)).isFailure()
```

Comment:
You shouldn't need to add wrappers for these. `assertFailue` should exist in assertk and
```kotlin
assertThat { ... }.isSuccess()
```
can be replaced with just
```kotlin
assertThat(...)
```

Reply:
I think that was deprecated

Reply:
Oh nevermind, I see what you're saying